gap/PolygonalComplexes/graphs.gd

```diff
@@ -850,5 +850,48 @@ DeclareAttribute( "FaceDigraphsGraph", IsPolygonalSurface );
 DeclareAttribute( "FaceNautyGraph", IsPolygonalSurface );
 #! @EndGroup
 
-
-
+#! @BeginGroup AllSimplicialSurfacesOfGraph 
+#! @Description 
+#! Return all (vertex-faithful) simplicial surfaces, that have <K>digraph</K> as face graph. 
+#! If <K>digraph</K> is not a face graph of a (vertex-faithful) simplicial surface, the empty list is returned.
+#! The parameter <K>vertexfaithful</K> indicates whether only vertex-faithful simplicial surfaces are searched. 
+#! The parameter <K>vertexfaithful</K> is by default false.
+#! <K>digraph</K> must be a cubic, connected, symmetric and simple digraph. The vertices of a simplicial 
+#! surface are certain cycles in the face graph. This method searches possible combinations of cycles, 
```

Comment:
Maybe better: The vertices of a simplicial surface can be identified with certain cycles in the face graph.